handlers/github_organization_repo_create_v1/readme.md

```diff
@@ -1,13 +1,13 @@
-== GitHub Organization Repository Creation
+# GitHub Organization Repository Creation
   Creates Repository to a Github organization.
-=== Parameters
-[name]
-  Name of desired repository user wishes to create.
-[org_name]
-  Name of organization of which you wish to create a new repository in.
+## Parameters
+  * [name]
+    * Name of desired repository user wishes to create.
+  * [org_name]
+    * Name of organization of which you wish to create a new repository in.
 
-=== Info
-[access_token]
+### Info
+* [access_token]
   As of 2/26/1016, Access Tokens using GitHub's oauth2 in an organization requires approval than a callbackURL, where the access token is sent to.
```

Comment:
This wasn't part of your changes, but the date referenced for access_token is off by a 1000 years. Maybe just fix that with your changes.

Reply:
Sorry Jack,  I changed the for format of the readme.md file.  It was originally just README without the md extension and the file didn't conform to markdown spec.  I changed it so that it would look nicer when viewed from the github UI.  The date was a hold over from the original code base.  I'll get it corrected in the next version.

Reply:
corrected date